src/UnitTest/Python/test_core.py

```diff
@@ -536,6 +536,19 @@ def test_reduction_sum(dim, keepdim):
     np.testing.assert_allclose(o3_dst.numpy(), np_dst)
 
 
+@pytest.mark.parametrize(
+    "dim",
+    [0, 1, 2, (), (0,), (1,), (2,), (0, 1), (0, 2), (1, 2), (0, 1, 2), None])
+@pytest.mark.parametrize("keepdim", [True, False])
+def test_reduction_mean(dim, keepdim):
+    np_src = np.array(range(24)).reshape((2, 3, 4)).astype(np.float32)
+    o3_src = o3d.Tensor(np_src)
+
+    np_dst = np_src.mean(axis=dim, keepdims=keepdim)
+    o3_dst = o3_src.mean(dim=dim, keepdim=keepdim)
```

Comment:
can we also test o3_src.cuda().mean (if we have cuda?)

Reply:
Done. Also added this to other python tests.